### PR TITLE
[Bref] Restart after uncaught exception

### PR DESCRIPTION
**Pull request by Nyholm**
Follow up after https://github.com/brefphp/bref/pull/1013

### CODE REVIEW

File: src/bref/src/BrefRunner.php [hunk 1]
```diff
@@ -31,7 +31,15 @@ public function run(): int
...
             }
-            $lambda->processNextEvent($this->handler);
+
+            /**
+             * In case the execution failed, we force starting a new process regardless
+             * of $this->loopMax. This is because an uncaught exception could have
+             * left the application in a non-clean state.
+             */
+            if (!$lambda->processNextEvent($this->handler)) {
+                return 1;
```
Comment (GrahamCampbell) [inline src/bref/src/BrefRunner.php:41]: I think bref chose to return zero in the same scenario?

Comment (Nyholm) [reply src/bref/src/BrefRunner.php]: Yes, Bref will always exit the application with code 0. Im not sure why. I'll research if there is a difference from AWS's point of view

Comment (Nyholm) [reply src/bref/src/BrefRunner.php]: Hm.. I didnt find anything interesting... Changed to exit with 0

Comment (GrahamCampbell) [reply src/bref/src/BrefRunner.php]: I guess it's because the daemon process didn't fail. It just decided to stop due to a subtask failing?

**Pull request merged**